device_backends/mmio/include/DeviceFile.h

```diff
@@ -11,18 +11,22 @@ namespace ChimeraTK {
   // Simple RAII wrapper for a device file
   class DeviceFile : private boost::noncopyable {
    private:
-    std::string _strerror(const std::string& msg) const;
-    const std::string _path;
-    int _fd;
+    [[nodiscard]] std::string _strerror(const std::string& msg) const;
```

Comment:
Our coding guide says public comes first and private last...

Reply:
it is public first/private last?